src/WebApi/WebApi/Extensions/RangeValidationAttributeExtension.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace WebApi.Extensions
+{
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using System.Linq;
+
+    public class RangeValidationAttributeExtension : ValidationAttribute
+    {
+        public int Min { get; set; }
+        public int Max { get; set; }
+
+        public RangeValidationAttributeExtension()
+        {
+            Min = 0;
+            Max = int.MaxValue;
+        }
+
+        public override bool IsValid(object value)
+        {
+            if (value is IEnumerable<int> list && list.Count() > 0)
```

Comment:
I would recommend using here `Any()` rather than `Count() > 0`, so we don't have to enumerate the collection just to check if it contains any elements.

```suggestion
            if (value is IEnumerable<int> list && list.Any())
```

Reply:
@TomasNiemczyk98 Thanks you, much appreciated , will do !!